data/strings/strings.txt

```diff
@@ -22904,6 +22904,141 @@
     zh-Hans = 加载地图
     zh-Hant = 載入地圖
 
+  [pref_tts_speedcams_auto]
+    tags = android
+    en = Warn if there is a risk of exceeding the speed limit
+    af = Waarsku indien daar ’n risiko is dat die grens oorskry word
+    ar = حذرني  إذا هناك احتمالية تجاوز الحد الأقصى للسرعة
```

Comment:
@organicmaps/translations please check if the strings below are correctly translated in your language. Thanks in advance!

Reply:
Romanian looks good.